lib/mix/lib/mix/tasks/new.ex

```diff
@@ -445,8 +445,8 @@ defmodule Mix.Tasks.New do
     use ExUnit.Case
     doctest <%= @mod %>
 
-    test "the truth" do
-      assert 1 + 1 == 2
+    test "salutates the world" do
+      assert <%= @mod %>.hello == :world
```

Comment:
@whatyouhide changed it in https://github.com/elixir-lang/elixir/pull/6159/commits/bef2c1603f35efdc46d1809935af7985b5e7a4dd, but the parenthesis make the tests fail for some reason and I can't find out why.

Could you give me a hand here?

Thanks!